EIPS/eip-###.md

```diff
@@ -0,0 +1,64 @@
+---
+eip: <to be assigned>
+title: Chain-specific addresses
+description: A standard for displaying CAIP-10 account identifiers in a human readable format
+authors: Lukas Schor (@lukasschor), Richard Meissner (@rmeissner), Pedro Gomes (@pedrouid), ligi <ligi@ligi.de> 
+discussions-to: https://ethereum-magicians.org/t/chain-specific-addresses/6449
+status: Draft
+type: Standards Track
+category: ERC
+created: 2021-08-26
+requires: CAIP-2, CAIP-3, CAIP-10
+---
+
+## Abstract
+This EIP introduced a new address standard to be adapted by wallets and dApps to display chain-specific addresses by mapping human-readable names to CAIP-3 blockchain IDs.
+  
+## Motivation
+The need for this EIP emerges from the increasing adoption of non-Ethereum Mainnet chains that use the Ethereum Virtual Machine. In this context, addresses become ambiguous, as the same address may refer to an EOA on chain X or a smart contract on chain Y. This will eventually lead to Ethereum users losing funds due to human error. For example, users sending funds to a smart contract wallet address which was not deployed on a particular chain.
+
+[CAIP-10](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-10.md) introduces an account identifier that encodes a CAIP-2 blockchain ID as part of the address. For EVM-based chains, these blockchain IDs are defined in [CAIP-3](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-3.md) and leverage [EIP-155](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-155.md) chainIDs. However, CAIP-10 targets developers, not end-users. These account identifiers are not meant to be displayed to users in dApps or wallets, and they were optimized for developer interoperability, rather than human readability.
+
+To solve the initial problem of user-facing addresses being ambiguous in a multichain context, we need to extend CAIP-10 with a user-facing format of displaying these account identifiers.
```

Comment:
@pedrouid Might be better to answer this, but how I understand this: 

CAIP-10 aims to standardize chain-specific identifiers across multiple blockchain ecosystems and should therefore only use a source of truth for blockchain identifiers that involve the least amount of consensus within the ecosystems. These are the chainIDs defined in EIP155, BIP122, etc. So to reduce any room for interpretations, CAIP-10 had to fall back to a format that is not ideal from a UX perspective but at least maximizes technical interoperability. Now we could try to establish the current vanilla CAIP-10 format, but it was not designed as a user-facing format (e.g. non-human readable), so that would likely be challenging. Thus this EIP tries to solve this via human-readable aliases that can be used user-facing and can then be resolved to CAIP-10 blockchain IDs. Other ecosystems might want to solve this differently (e.g. adopting CAIP-10 directly or having a different alias format).

Reply:
This discussion is very related to the discussion over at https://ethereum-magicians.org/t/caip-2-blockchain-references/3612/13, lets continue there since that is a better place for long-form discussion.

Reply:
> IMO, CAIP-10 should be adjusted to be one name per network

honestly that's exactly what EIP-3370 is solving and the ethereum-lists/chains maps a shortName to a EIP-155 chainId which is described as CAIP-2 chainId

if it's so controversial to use CAIPs inside EIPs then we could just change this proposal to map shortNames to EIP-155 chainIds

but I think it's very odd that CAIP-2 is that controversial when all it does is prefix an integer with `eip155:` to specify what the meaning of the integer